src/coreclr/jit/gentree.h

```diff
@@ -7643,22 +7697,26 @@ inline unsigned int GenTree::GetRegSpillFlagByIdx(int regIndex) const
 
 #endif // FEATURE_MULTIREG_RET
 
+#if 0 // REVIEW
 #if defined(TARGET_XARCH) && defined(FEATURE_HW_INTRINSICS)
     if (OperIs(GT_HWINTRINSIC))
     {
         // At this time, the only multi-reg HW intrinsics all return the type of their
         // arguments. If this changes, we will need a way to record or determine this.
         assert(TypeGet() == TYP_STRUCT);
+        // REVIEW: what should this return?
         return gtGetOp1()->TypeGet();
     }
 #endif
+#endif // 0
```

Comment:
This looks like a copy-paste mishap from `GetRegTypeByIndex`. I guess all callers would be false anyway as they are checking for `GTF_SPILL = 0x80` flag, so it can probably just be removed.